### PR TITLE
Add libopenni-dev to docker dependencies and fix sha for drake-iiwa-driver

### PR DESCRIPTION
**Pull request by peteflorence**
This should maybe be placed somewhere else, but currently `master` cannot finish top-level `spartan` build without it.

---
This change is [Reviewable](https://reviewable.io/reviews/robotlocomotion/spartan/160)

### CODE REVIEW

**Comment by gizatt** [comment]
:lgtm: Though I'm pretty confused why it's not being brought in by any of our enormous dependencies (namely Director...). It's the kind of thing that ought to be installed on our system, anyway, so I don't mind it.

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[setup/docker/install_dependencies.sh, line 14 at r2](https://reviewable.io:443/reviews/robotlocomotion/spartan/160#-KwGY2yqiID8hvviguxU:-KwGY2yqiID8hvviguxV:bqie7wk) ([raw file](https://github.com/robotlocomotion/spartan/blob/203cdb596544efed04705943acb7b845b53d5582/setup/docker/install_dependencies.sh#L14)):*
> ```Shell
>   openssh-client \
>   unzip \
>   htop \
> ```

FYI I added a line continuation here that was missing

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/spartan/160#-:-KwGY7U2Rhm-f7wO5wOc:b6h681x)*

**Comment by peteflorence** [comment]
This PR also includes drake-iiwa-driver sha repoint

**Comment by gizatt** [comment]
:+1: 

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/spartan/160#-:-KwGhlMcD5kfv2gYBywj:b-8fx2u7)*

**Comment by gizatt** [comment]
Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[modules/spartan/perception/handeyecalibration.py, line 33 at r4](https://reviewable.io:443/reviews/robotlocomotion/spartan/160#-KwGkPIS2OUYVj3wCKYR:-KwGkPIS2OUYVj3wCKYS:b-t5ljd2) ([raw file](https://github.com/robotlocomotion/spartan/blob/734830ceb9cc7f4dddee12fe9e0ecf27cb591908/modules/spartan/perception/handeyecalibration.py#L33)):*
> ```Python
> 	from labelfusion.cameraposes import CameraPoses
> except:
> 	print "Have you sourced setup_environment_spartan.sh in src/LabelFusion?"
> ```

should it fail here?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/spartan/160)*

**Comment by peteflorence** [comment]
thanks @gizatt, yes it should have a quit() on the except

File: modules/spartan/perception/handeyecalibration.py [hunk 1]
```diff
@@ -31,8 +31,7 @@
 	from labelfusion.cameraposes import CameraPoses
 except:
 	print "Have you sourced setup_environment_spartan.sh in src/LabelFusion?"
```
Comment (manuelli) [inline modules/spartan/perception/handeyecalibration.py]: I actually moved this file out of LabelFusion and to `spartan/config/setup_environment_labelfusion.sh`. I think it's better to have in `spartan` rather that in `LabelFusion`.

File: setup/docker/install_dependencies.sh [hunk 2]
```diff
@@ -11,4 +11,5 @@ apt install --no-install-recommends \
   git \
   openssh-client \
   unzip \
-  htop
+  htop \
+  libopenni-dev
```
Comment (manuelli) [inline setup/docker/install_dependencies.sh]: can we make an issue to track down why this is necessary.

Comment (peteflorence) [reply setup/docker/install_dependencies.sh]: good call, made one

File: modules/spartan/perception/handeyecalibration.py (hunk 1 above)
Comment (peteflorence) [reply modules/spartan/perception/handeyecalibration.py]: :+1: adjusted the print

**Pull request merged**